ci/Jenkinsfile.macos

```diff
@@ -160,3 +159,11 @@ def getArch() {
       if (tokens.contains(arch)) { return arch }
     }
 }
+
+def getQTPath() {
+    if(getArch()=='x86_64') {
+      return '/usr/local/qt/5.15.2/clang_64'
+    } else {
+      return '/opt/homebrew/opt/qt@5'
+    }
+}
```

Comment:
But why tho? The `aqtinstall` from Brew just behaves differently on different OSes?

Reply:
I saw in our `ansible` setup, we only install QT via this script in `status-desktop-setup` role, which is invoked only for `x86_64` hosts for some unknown reason https://github.com/status-im/infra-ci/blob/master/ansible/slaves.yml#L34

Reply:
Actually we don't need `aqt`, we can use brew in both architectures for MacOS.

Reply:
also the output of `$(brew --prefix)` gives us a way to generate the path to `QT` dynamically.
`homebrew` is installed on different locations in ARM mac vs Intel mac.